src/site/topic23.rst

```diff
@@ -277,6 +277,56 @@ Max & Remove Max
 General Remove
 --------------
 
+* A general remove is a little more complex than the remove min or max
+
+.. image:: img/binarysearchtree_removeuh.png
+   :width: 500 px
+   :align: center
```

Comment:
Think the complexity would be illustrated better if there were multiple child nodes on the node being removed, it's pretty clear in this example you'd just do `left.parent = this.parent`.